hermeto/core/package_managers/yarn_classic/main.py

```diff
@@ -31,6 +31,7 @@
 
 
 MIRROR_DIR = "deps/yarn-classic"
+YARN_NETWORK_TIMEOUT_MILLISECONDS = 600000
```

Comment:
Just curious: what is the default value here?

Reply:
30 seconds (30000 ms)

Reply:
Can we please accompany this constant value with some reasoning on what type of error issue we've faced with the "more optimistic" default timeout?